src/core/server.js

```diff
@@ -676,6 +676,17 @@ Server.prototype._handleResponse = function(message) {
     const result = message.result;
     const responseEvent = 'response_' + command;
 
+    if (command === 'submit' && result.engine_result === 'tefPAST_SEQ' &&
```

Comment:
Why not `tefALREADY`? Isn't it unsafe to convert to success for a `tefPAST_SEQ`?